.github/workflows/run_analysis.yml

```diff
@@ -166,19 +123,19 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Download Data
-        uses: docker://pgc-images.sbgenomics.com/d3b-bixu/open-pedcan:analysisjob
+        uses: docker://pgc-images.sbgenomics.com/d3b-bixu/open-pedcan:latest
```

Comment:
Using the `latest` tag for analysis means that changes to the *Dockerfile* and changes to the packages/data **HAVE** to be separate PRs.